config/board.ini

```diff
@@ -270,3 +270,43 @@ lib_ignore =
 extra_scripts =
     ${config:smallNoI2s.extra_scripts}
     pre:./scripts/get_git_rev.py
+
+
+; ********************************************************************************
+; M5Stack Core ESP32 WiFi and Bluetooth Module Development Board
+; ********************************************************************************
+[board:m5stack_core]
+extends = mcu:esp32, display:m5stack_core, config:normal
+board = m5stack-grey
+board_build.partitions = default_8MB.csv
+board_build.filesystem = littlefs
+build_flags =
+    ${mcu:esp32.build_flags}
+    ${display:m5stack_core.build_flags}
+    -DBOARD_M5STACK 

```

Comment:
Is it necessary somewhere?

Reply:
In your current master there is an if statement for pin definition and this was the parameter (in the current dev branch it is not needed any more and gone in board.h)